### ISSUE TITLE
[Request] lock position of alphabet and apps on drawer

### ISSUE THREAD

**Issue by JONI350** [8bitPit/Niagara-Issues#1762]
### Is your feature request related to a problem?

Somewhat a problem. When using the drawer sometimes u need to do a bit of juggling or go back and forth to reach a letter u want because the alphabet is always on the same spot on screen

It is also a bit inconsistent if u do have fewer apps on it making the position of the apps change 

This is the inconsistency im talking about, it behaves different, idk if its intentional or not but i think its anti intuitive (also when u drop the alphabet the apps stay exactly where u see on screen tho)

Fewer amount of apps

https://user-images.githubusercontent.com/25126715/144186277-7743e4e4-7d44-4b91-befe-b90f15da3f1c.mp4

### Describe the solution you'd like

Basically my suggestion is that u click up here (and also on the niagara config obviously) 

![Screenshot_20211201-035752_Niagara Launcher](https://user-images.githubusercontent.com/25126715/144186918-fa583061-6e75-4850-8877-709d3ac87846.jpg)

and hold so they open a menu with those

![Polish_20211201_040118394](https://user-images.githubusercontent.com/25126715/144187189-2ce067be-ec42-49a2-896f-0590ba63457f.jpg)

To adjust the height of apps and alphabet, maybe one for each tho

**Comment by Maxr1998** [comment]
The position of the apps when just a few are installed has technical as well as design reasons - it would create a large empty space at the end of the list that'd look and feel weird. I think that was reported already, but I currently cannot find the corresponding issue, sorry.

The scrollbar adapts and is sized according to your screen dimensions and the amount of apps/categories on your screen, we don't think a customization option for that is necessary.
